packages/text-field/theme/lumo/vaadin-input-field-shared-styles.js

```diff
@@ -0,0 +1,252 @@
+import { registerStyles, css } from '@vaadin/vaadin-themable-mixin/register-styles.js';
```

Comment:
The theme files have no license header. Was like that in the old version, but may be an oversight.

Reply:
Added the license headers.